apm-agent-plugins/apm-profiling-plugin/src/main/java/co/elastic/apm/agent/profiler/ProfilingConfiguration.java

```diff
@@ -54,6 +54,15 @@ public class ProfilingConfiguration extends ConfigurationOptionProvider {
         .tags("added[1.15.0]", "experimental")
         .buildWithDefault(false);
 
+    private final ConfigurationOption<Boolean> disableProfilingLogging = ConfigurationOption.<Boolean>booleanOption()
+        .key("profiling_logging_disabled")
+        .configurationCategory(PROFILING_CATEGORY)
+        .description("By default, async profiler prints warning messages about missing JVM symbols to standard output. \n" +
+            "Set this option to `true` to suppress such messages")
+        .dynamic(true)
+        .tags("added[1.37.0]")
+        .buildWithDefault(false);
```

Comment:
[minor] maybe better to keep a positive wording here, so `profiling_logging_enabled` and make the default value `false`.

Reply:
I started with `enabled`, but I figured it's better to reflect the opt-in-for-special-cases nature of this config with `disabled`. I am not sure...

Reply:
Almost all the other configuration options that we have are `_enabled` suffix, thus that would be at least consistent with others:
- `circuit_breaker_enabled`
- `span_compression_enabled`
- `profiling_inferred_spans_enabled`